individualForm.js

```diff
@@ -1,159 +1,208 @@
-export default `
+module.exports = `
+# Now hand-edited
+
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix owl: <http://www.w3.org/2002/07/owl#>.
 @prefix ui: <http://www.w3.org/ns/ui#>.
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#>.
 @prefix : <#>.
 
-<http://www.w3.org/2006/vcard/ns#Individual>
-    ui:creationForm <#form1> .
+# Ontology additions or interpretations needed for the form to work well
+
+# The ontolgy file doesnt make them disjoint.  This makes the selctor be a choice.
+vcard:TelephoneType   owl:disjointUnionOf ( vcard:Cell vcard:Home vcard:Work) .
+vcard:Type   owl:disjointUnionOf (vcard:Home vcard:Work) . # for email
+
+# Better field labels
+vcard:Cell ui:label "mobile"@en . # app will imake nitial caps if nec
```

Comment:
initial caps in a phone number?

```suggestion
vcard:Cell ui:label "mobile"@en . # app will make initial caps if nec
```